stable/external-dns/templates/_helpers.tpl

```diff
@@ -428,3 +428,14 @@ external-dns: transip.apiKey
     Please set the apiKey parameter (--set transip.apiKey="xxxx")
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the service account name.	Return the service account name used by the pod.
+*/}}
+{{- define "serviceaccount.name" -}}
```

Comment:
I would change this to, by default, let it create the service account, and, if `.Values.rbac.serviceAccount.name` is provided then use the existing one and not create a new one.